fizzbuzz/fizzbuzz.c

```diff
@@ -10,9 +10,28 @@
 */
 int fizzbuzz(int n)
 {
-
-
-}
+    int counter = 0;
+    for (int i = 0; i < n; i++)
+    {
+        if (i % 3 == 0)
+        {
+            printf("Fizz\n");    
+        }
+        else if (i % 5 == 0)
+        {
+            printf("Buzz\n");
+        }
+        else if (i % 3 == 0 && i % 5 == 0)
```

Comment:
This needs to go in the first if statement of your for loop. If it's not up there when i gets to a number that's divisible by 3 and 5, like 15 it will not print out "Fizzbuzz" as intended because the `if (i % 3 == 0)` condition will trigger and then the loop will just go on to the next iteration. Everything else looks good though!

Reply:
Ah... good point! Thanks!